modules/plugins/languages/asm.nix

```diff
@@ -20,12 +32,11 @@ in {
     };
 
     lsp = {
-      enable = mkEnableOption "Assembly LSP support (asm-lsp)" // {default = config.vim.lsp.enable;};
-
-      package = mkOption {
-        type = package;
-        default = pkgs.asm-lsp;
-        description = "asm-lsp package";
+      enable = mkEnableOption "Assembly LSP support" // {default = config.vim.lsp.enable;};
+      servers = mkOption {
+        description = "Assembly LSP server to use";
+        type = listOf (enum (attrNames servers));
+        default = defaultServers;
```

Comment:
Please keep the ordering as

```
2. type
2. default
3. (optional) example
4. description
```

Reply:
done [e0cd56f](https://github.com/NotAShelf/nvf/pull/913/commits/e0cd56f39e218546d8258bd838754e06df64d792)